cmd/service.go

```diff
@@ -159,6 +160,8 @@ func runServiceStartCmd(cmd *cobra.Command, args []string) {
 		}
 	}()
 
+	statushooks.Show(ctx)
```

Comment:
Can we try to factor this so that as far as possible we do
```
func Something(){
    statushooks.Show(ctx)
     defer statushooks.Done(ctx)
...
}
```